Add issue templates (#58)
latticesurgery-com/web-ui: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Create a report to help us improve the compiler
title: "[BUG]"
labels: bug
assignees: ''

---

Issue Description
-----------------

Insert a short description of the bug here, along with what you expected the behavior to be.

How to Reproduce
----------------

> If possible, please provide us with the steps that lead to the behavior that you are seeing, and include the snippet of the code that generates the error as well as the error message from python below. 

Author: ```suggestion
If possible, please provide us with the steps that lead to the behavior that you are seeing, and include the snippet of the code that generates the error as well as the error message from the browser's or node's console below. 
```

### Code Snippet

> ```python

Author: ```suggestion
```ts
```

Include a snippet of the code that produces the error here.
```

### Error Output

```
Additionally, please copy and paste the output of the above code here.
```

> Environment Context
> -------------------
> 
> Please add information about your operating system, python version and dependencies here. 
> 
> ```
> Copy and paste the output of pip freeze or conda list here.
> ```

Reviewer: ```suggestion
```
I don't think we would need this part.

Reviewer: Also maybe add a screenshot section under the How to Reproduce section - I think that is more important for web applications.
